manager/state/raft/storage.go

```diff
@@ -248,5 +249,5 @@ func (n *Node) restoreFromSnapshot(data []byte, forceNewCluster bool) error {
 		n.cluster.ClearMember(member)
 	}
 
-	return nil
+	return snapshot.Membership.Members, nil
```

Comment:
I have a preference for returning `snapshot.Membership` and letting the caller handle removal as well, since it's a bit inconsistent to have this function handle removal but not addition. It's not a big deal though.